.travis.yml

```diff
@@ -91,12 +91,15 @@ script:
   # Run tests that don't leverage specific ML frameworks
   - pytest --cov=mlflow --verbose --large --ignore=tests/h2o --ignore=tests/keras
     --ignore=tests/pytorch --ignore=tests/pyfunc --ignore=tests/sagemaker --ignore=tests/sklearn
-    --ignore=tests/spark --ignore=tests/tensorflow
+    --ignore=tests/spark --ignore=tests/tensorflow --ignore=tests/onnx
   # Run ML framework tests in their own Python processes. TODO: find a better method of isolating
   # tests.
   - pytest --verbose tests/h2o --large
   # TODO(smurching): Re-enable Keras tests once they're no longer flaky
   # - pytest --verbose tests/keras --large
+  - if [[ "$TRAVIS_PYTHON_VERSION" == "3.6" ]]; then
```

Comment:
I don't think that this is necessary. The `pytestmark` setting in the test suite should automatically skip the tests when running against Python 2.

Reply:
I removed the pytest mark from the tests and kept the if statement in travis scripts.

Reply:
Added back the `pytestmark `  and removed the if condition from travis